src/ralph/data_center/models/physical.py

```diff
@@ -602,6 +602,18 @@ def _validate_slot_no(self):
                         'Slot number cannot be filled when asset is not blade'
                     )
                 })
+            if self.parent:
+                dc_asset_with_slot_no = DataCenterAsset.objects.filter(
+                    parent=self.parent, slot_no=self.slot_no
+                ).exclude(pk=self.pk).first()
+                if dc_asset_with_slot_no:
+                    raise ValidationError({
+                        'slot_no': (
+                            'Slot is already occupied by: {}'.format(
+                                dc_asset_with_slot_no
```

Comment:
actually it'd be useful to have link here to the dc asset occupying this slot (just like for mgmt ip/hostname)